apps/api/app/identity_api.py

```
@@ -615,10 +715,13 @@ def okta_sso_callback(
 def scim_list_users(
     session: SessionDep,
     authorization: Annotated[str | None, Header(alias="Authorization")] = None,
-    filter: str | None = Query(default=None),
+    filter: Annotated[str | None, Query()] = None,  # pylint: disable=redefined-builtin
```

Ruff (0.15.15)

[error] Function argument `filter` is shadowing a Python builtin

(A002)